sqlalchemy_jsonapi/serializer.py

```diff
@@ -459,7 +459,7 @@ def _check_instance_relationships_for_delete(self, instance):
             check_permission(instance, rel_key, Permissions.EDIT)
 
             if rel.cascade.delete:
-
+                # pragma: no cover
```

Comment:
I think you should create a model that does delete on cascade in order to test this. Marking things as "no cover" is for cases that literally *can't* be tested for technical reasons.